### PR TITLE
Fix the config.php owner check for console.php

### PR DESCRIPTION
**Pull request by phlb**
The config directory must be taken from OC::$configDir

The config directory was hardcoded in console.php.

### CODE REVIEW

File: console.php [hunk 1]
```diff
@@ -31,7 +31,7 @@
 			exit(0);
 		}
 		$user = posix_getpwuid(posix_getuid());
-		$configUser = posix_getpwuid(fileowner(OC::$SERVERROOT . '/config/config.php'));
+		$configUser = posix_getpwuid(fileowner(OC::$configDir . "/config.php" ));
```
Comment (Xenopathic) [inline console.php]: Please use single quotes for the literal string part. Also, `$configDir` already has the trailing slash, so you might need to remove that from the literal string.

**Comment by MorrisJobke** [comment]
cc @DeepDiver1975

**Comment by DeepDiver1975** [comment]
This change has not one single benefit - configDir is always(within the scope of this script) pointing to SERVERROOT . '/config'

**Comment by Xenopathic** [comment]
@DeepDiver1975 But on the other hand, it does make the intent a little more obvious, and pre-empts any possible-potential bugs we might at some point conceivably-prospectively have. Yes, it's unlikely we will break this, but the possibility is still there :smile:

**Comment by DeepDiver1975** [comment]
I'm willing to accept this change if @phlb will respond to the comments above

**Comment by phlb** [comment]
Ok for the modification.

@DeepDiver1975 the configdir depends on lib/base.php. In several cases, the config dir is not ``OC::$SERVERROOT . '/config/'``. Currently only for testing purposes but also for the PR #16424.

**Comment by Xenopathic** [comment]
:+1:

**Comment by MorrisJobke** [comment]
:+1:

**Comment by MorrisJobke** [comment]
I will port this to directly to master and close this PR, because it is against a stable version.

**Comment by MorrisJobke** [comment]
#17546

**Pull request closed without merging**